### PR TITLE
#156: Properly parse models from 'getAll' in Java ModelServerClient

### PR DESCRIPTION
**Pull request by martin-fleck-at**

### CODE REVIEW

File: examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java [hunk 1]
```diff
@@ -149,6 +155,24 @@ private static void handleGet(final ModelServerClient client, final String[] com
...
+         System.out.println(response.body().size() + " models");
+         for (Model<String> model : response.body()) {
+            System.out.println("- " + model.getModelUri() + ": " + model.getContent());
+         }
+      } else {
+         Response<List<Model<EObject>>> response = client.getAll(format).join();
+         System.out.println(response.body().size() + " models");
+         for (Model<EObject> model : response.body()) {
+            System.out.println("- " + model.getModelUri() + ": " + model.getContent().eClass());
+         }
```
Comment (ndoschek) [inline examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java:172]: Why do we need the distinction here? If the `xmi` model content is the same string output as the `eClass` output in my case.

**Review by ndoschek — CHANGES_REQUESTED** [review]
Thanks a lot for the fix Martin! Looks good to me!
I do have a minor comment (please see inline) and one small request: Could you please update the year ranges in the file headers? Thanks!

File: examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java (hunk 1 above)
Comment (martin-fleck-at) [reply examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java:172]: I'm not sure what you mean by the same content but I split this method to showcase the difference between the json case where you get "only" the string and the 'xmi' where you get the actual EObject and thus can access the eClass.

Comment (ndoschek) [reply examples/org.eclipse.emfcloud.modelserver.example/src/main/java/org/eclipse/emfcloud/modelserver/example/client/ExampleModelServerClient.java]: Sorry for the confusion, I overlooked the format argument difference here.

**Pull request merged**